ivy/functional/ivy/device.py

```diff
@@ -551,8 +551,19 @@ def set_default_device(device: Union[ivy.Device, ivy.NativeDevice]):
     default_device_stack.append(device)
 
 
-def unset_default_device():
-    """"""
+def unset_default_device()-> None:
+    """Resets the default device to "cpu".
+
+    Examples
+    --------
+    >>> ivy.set_default_device("gpu:0")
+    >>> ivy.default_device()
+    device(type="gpu:0")
```

Comment:
Thanks for making the edits!
I just want to check the format of the output you're getting when running this example locally.
Is the format of your output showing as `device(type="gpu:0")` and not just `gpu:0` ?

![clip_2](https://user-images.githubusercontent.com/72862115/174776057-2942209a-dfae-422b-99f9-62e3c49286fc.png)